packages/module/src/CullingInfo/CullingInfo.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { Tooltip, TooltipProps } from '@patternfly/react-core';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
+import { CullingDate, CullingInfo, calculateTooltip } from './CullingInfoUtils';
+
+export type Render = (config: { msg: string }) => React.ReactElement<any, any> | null;
+
+const useStyles = createUseStyles({
+  inventoryCullingWarning: {
+    color: 'var(--pf-v6-global--warning-color--200)',
+    fontWeight: 'var(--pf-v6-global--FontWeight--bold)',
+    svg: {
+      marginRight: 'var(--pf-v5-global--spacer--sm)'
+    }
+  },
+  inventoryCullingDanger: {
+    color: 'var(--pf-v6-global--warning-color--200)',
+    fontWeight: 'var(--pf-v6-global--FontWeight--bold)',
+    svg: {
+      marginRight: 'var(--pf-v6-global--spacer--sm)'
+    }
+  }
+});
+
+export interface CullingInformation extends Omit<TooltipProps, 'content'> {
```

Comment:
```suggestion
/** extends TooltipProps */
export interface CullingInformation extends Omit<TooltipProps, 'content'> {
```

this displays the base interface also in the docs
![Snímek obrazovky 2025-01-28 v 16 15 37](https://github.com/user-attachments/assets/f1a5467b-4eda-4b06-b8bd-08fdc4b395d2)

Reply:
fixed